front/src/modules/trainschedule/components/ImportTimetableItem/helpers/importTimetableItem.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { describe, it, expect } from 'vitest';
+
+import type { CichDictValue, Step } from 'applications/operationalStudies/types';
+
+import { buildSteps } from './BuildSteps';
+
+describe('buildSteps', () => {
+  it('increments the day offset when the arrival time is before the previous departure', () => {
+    const ocpTT1 = {
```

Comment:
The test is not really comprehensible for humans.
You can use some xml string directly as the test input.

for example something like this:

```
const xmlString = `
<library>
  <book>
    <title>The Hobbit</title>
    <author>J.R.R. Tolkien</author>
    <year>1937</year>
  </book>
  <book>
    <title>1984</title>
    <author>George Orwell</author>
    <year>1949</year>
  </book>
</library>
`;

const parser = new DOMParser();
const xmlDoc = parser.parseFromString(xmlString, "text/xml");
const result = xmlDoc.getElementsByTagName("book")

expect(result.length).toBe(2)
```